### PR TITLE
Add help files for classification

### PR DESCRIPTION
**Pull request by koenderks**
+ make some layout changes in cluster help files

### CODE REVIEW

**Comment by koenderks** [comment]
After this only 2 help files remaining: random forest clustering (Joran is making) and regularized linear regression (Tim)

**Comment by TimKDJ** [comment]
@MyrtheV could you read these over?

**Comment by MyrtheV** [comment]
I will try to look at these today.

File: Resources/Help/analyses/mlregressionboosting.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+
+### Assumptions
+- The predictors consist of continuous, nominal, or ordinal variables.
+- The target is a continuous variable.
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered.
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlregressionboosting.md]: In this box, the variables...

File: Resources/Help/analyses/mlregressionboosting.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+- Add generated indicator to data: Add the generated test set indicator from the option above to your data set. Requires a column name.
+- Test set indicator: Use an indicator variable to select data for the test set.
+
+#### Training and Validation Data
+- Sample *x*% for validation data: Randomly sample a percentage from the remaining data (after selecting the test set).
+- K-fold with *k* folds: Partition the remaining data in *k* parts.
+
+### Training Parameters 
+#### Algorithmic Settings
+- Shrinkage: a shrinkage parameter applied to each tree in the expansion. Also known as the learning rate or step-size reduction 0.001 to 0.1 usually work, but a smaller learning rate typically requires more trees.
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlregressionboosting.md]: -Shrinkage: A shrinkage...

File: Resources/Help/analyses/mlregressionknn.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+
+### Assumptions
+- The predictors consist of continuous, nominal, or ordinal variables.
+- The target is a continuous variable.
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered.
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlregressionknn.md]: In this box, the variables

File: Resources/Help/analyses/mlregressionrandomforest.md [hunk 4]
```diff
@@ -1,3 +1,77 @@
...
+
+### Assumptions
+- The predictors consist of continuous, nominal, or ordinal variables.
+- The target is a continuous variable.
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered.
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlregressionrandomforest.md]: In this box, the variables...

File: Resources/Help/analyses/mlclassificationboosting.md [hunk 5]
```diff
@@ -1,3 +1,83 @@
 Boosting Classification
 ==========================
 
+Boosting works by sequentially adding predictors to an decision tree ensemble, each one correcting its predecessor. However, instead of changing the weights for every incorrect classified observation at every iteration, Boosting method tries to fit the new predictor to the residual errors made by the previous predictor.
+
+### Assumptions
+- The predictors consist of continuous, nominal, or ordinal variables.
+- The target is a nominal or ordinal variable
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlclassificationboosting.md]: The ... variable.

File: Resources/Help/analyses/mlregressionboosting.md [hunk 6]
```diff
@@ -1,3 +1,79 @@
...
+- Interaction depth: Integer specifying the maximum depth of each tree (i.e., the highest level of variable interactions allowed. A value of 1 implies an additive model, a value of 2 implies a model with up to 2-way interactions, etc. Default is 1.
+- Min. observations in node: Integer specifying the minimum number of observations in the terminal nodes of the trees. Note that this is the actual number of observations, not the total weight.
+- Training data used per tree: select the percentage of training data that is used to train each individual tree.
+- Loss function: The loss function used. Can be either *Gaussian*, *Laplace*, or *t*.
+- Scale variables: scales the variables. Standardization ensures that values of variables from different scales range into a specific similar scale. As a result, standardizing provides numerical stability, which improves the clustering output. JASP uses the Z-score standardization of a mean of 0 and a standard deviation of 1. This option is selected by default.
+- Set seed: gives the option to set a seed for your analysis. Setting a seed will exclude random processes influencing an analysis.
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlregressionboosting.md]: Would like to have this explained in more detail. For example, add what happens if you repeat analysis.

File: Resources/Help/analyses/mlclassificationboosting.md [hunk 7]
```diff
@@ -1,3 +1,83 @@
...
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered.
```
Comment (MyrtheV) [inline Resources/Help/analyses/mlclassificationboosting.md]: In this box, the...

**Comment by MyrtheV** [comment]
@koenderks The help files look good, but I have a few suggestions/questions: 

-The input options, as 'Plots' and 'Tables, are the first input options after the assignment box, but in the help files they come last. I would prefer to have them right after the assignment box, so that you keep the order in the analysis. 
-In the part 'output options' of the help files, there was only a description of 1 table. Descriptions of plots were, for example, missing. Do you have a specific reason to leave them out? 
-After the colon, you sometimes use capital letters and sometimes not (I assume for grammar). Because the help file is kind of a summation, I would prefer capital letters after the colon. However, this is just my preference, as I find that easier to read. 
-I could not find the example 'Students' Grades' yet. Does this still need to be added in the data library? Also, I could find the 'telcoCustomerChurn', however, I could not see the examples of the analyses with this data file. Does this still need to be added?

**Comment by koenderks** [comment]
In an update for PR

- The input options, as 'Plots' and 'Tables, are the first input options after the assignment box, but in the help files they come last. I would prefer to have them right after the assignment box, so that you keep the order in the analysis. 
**Done**
- In the part 'output options' of the help files, there was only a description of 1 table. Descriptions of plots were, for example, missing. Do you have a specific reason to leave them out?
**Yes, descriptions of plots and tables are given under Tables and Plots. Also, I do have a life besides this so, in light of that, we might make these descriptions some other time. For now, an extensive description of the main table should be enough.**
- After the colon, you sometimes use capital letters and sometimes not (I assume for grammar). Because the help file is kind of a summation, I would prefer capital letters after the colon. However, this is just my preference, as I find that easier to read.
**Done**
- I could not find the example 'Students' Grades' yet. Does this still need to be added in the data library? Also, I could find the 'telcoCustomerChurn', however, I could not see the examples of the analyses with this data file. Does this still need to be added?
**Is currently in JASP, also I changed 'example' to 'example data'**

**Pull request merged**